package.json

```diff
@@ -34,5 +35,8 @@
   },
   "devDependencies": {
     "eslint": "2.8.0"
+  },
+  "dependencies": {
+    "eslint-plugin-react": "6.2.0"
```

Comment:
My concern was that all projects would be downloading this plugin, but I guess that's not such a big deal since this package should be a dev dependency.